quad/cquads/cquads.go

```diff
@@ -45,8 +54,8 @@ func NewDecoder(r io.Reader) *Decoder {
 	return &Decoder{r: bufio.NewReader(r)}
 }
 
-// Unmarshal returns the next valid N-Quad as a quad.Quad, or an error.
-func (dec *Decoder) Unmarshal() (quad.Quad, error) {
```

Comment:
To clarify the naming. There is a sliceReader type at rw.go:51, for example, that is used in tests to load small quad sets. It will be weird to have an Unmarshal method on a slice. There is also a comment in dump.go about dumping the results of specific query only, so we might have a ReadQuad implementation that wraps the iterator too. And again, Unmarshal is a strange method for iterator.